adapters/repos/db/crud_integration_test.go

```diff
@@ -443,6 +452,44 @@ func TestCRUD(t *testing.T) {
 		assert.Equal(t, thingID, schema["uuid"], "has id in schema as uuid field")
 	})
 
+	t.Run("adding a thing", func(t *testing.T) {
```

Comment:
This test block has the same name (`"adding a thing"`) as a previous test in line 108. I think it would be nice if we could adjust the name to make explicit how this test block differs from the one in line 108 or what the intentions behind it are.